jetcd-core/src/main/java/io/etcd/jetcd/impl/ClientConnectionManager.java

```diff
@@ -174,7 +175,7 @@ ManagedChannelBuilder<?> defaultChannelBuilder(String target) {
             throw new IllegalArgumentException("At least one endpoint should be provided");
         }
         if (vertx == null) {
-            vertx = Vertx.vertx();
+            vertx = Vertx.vertx(new VertxOptions().setUseDaemonThread(builder.useDaemonThread()));
```

Comment:
although I appreciate you took backward compatibility into account, given this is mainly used as a library, I wonder if this should be the default behavior and avoid adding yet another configuration option.

Reply:
SGTM, that simplifies matters a lot and certainly helps us fix our issue.